tooling/lib/get-schema.js

```diff
@@ -9,11 +9,17 @@ module.exports = function getSchema(pathOperation, oas) {
   try {
     if (pathOperation.requestBody.content) {
       const type = Object.keys(pathOperation.requestBody.content)[0];
-      return { type, schema: pathOperation.requestBody.content[type].schema };
+
+      return {
+        type,
+        schema: pathOperation.requestBody.content[type],
```

Comment:
Had to slightly change how this library works because it was discarding `example` and `examples` by just returning the content type schema.